### PR TITLE
Add implementation of the quicksort algorithm

### PR DESCRIPTION
**Pull request by christosderetzis**

### CODE REVIEW

File: sorting_visualizer/sort.py [hunk 1]
```diff
@@ -78,8 +78,33 @@ def mergesort(A):
 
         counter = 2 * counter
 
-def quicksort(A):
-    return
+# quicksort is a recursive algorithm
+def quicksort(A, low, high):
```
Comment (debdutgoswami) [inline sorting_visualizer/sort.py]: This won't work. You need to create generators using `yield`. Also, note that you can only use iterative solutions. Recursive solutions don't exactly return generator objects at each iteration.

**Review by debdutgoswami — CHANGES_REQUESTED** [review]

File: sorting_visualizer/sort.py [hunk 2]
```diff
@@ -79,7 +79,70 @@ def mergesort(A):
...
+            stack[top] = p - 1
+
+        # If there are elements on right side of pivot,
+        # then push right side to stack
+        if p + 1 < high:
+            top = top + 1
+            stack[top] = p + 1
+            top = top + 1
+            stack[top] = high
+    yield A
```
Comment (debdutgoswami) [inline sorting_visualizer/sort.py]: This should be inside the while loop.

**Review by debdutgoswami — CHANGES_REQUESTED** [review]
Fixing this should be fine and ready to be merged. Great job.

File: sorting_visualizer/sort.py (hunk 2 above)
Comment (christosderetzis) [reply sorting_visualizer/sort.py]: I just fixed that

**Review by debdutgoswami — APPROVED** [review]

**Comment by debdutgoswami** [comment]
Thanks for your contribution.

**Pull request merged**